sql/npc_list.sql

```diff
@@ -30742,22 +30742,6 @@ INSERT INTO `npc_list` VALUES (17780996,'blank','     ',0,0.000,0.000,0.000,0,50
 INSERT INTO `npc_list` VALUES (17780997,'blank','     ',0,0.000,0.000,0.000,0,50,50,0,0,0,6,3,0x0000340000000000000000000000000000000000,0,NULL,1);
 INSERT INTO `npc_list` VALUES (17780998,'Sweepstox','Sweepstox',85,14.699,0.000,8.479,32789,50,50,0,0,0,0,27,0x0000550000000000000000000000000000000000,32,'ABYSSEA',1);
 INSERT INTO `npc_list` VALUES (17780999,'Treasure_Coffer','Treasure Coffer',85,13.699,0.000,9.310,1,50,50,0,4,36,0,3,0x0000C20300000000000000000000000000000000,0,'ABYSSEA',0);
-
-INSERT INTO `npc_list` VALUES (17781000,'blank','     ',57,-16.1250,0.675,5.4297,98305,40,40,0,0,64,2,2075,0x0000FC0200000000000000000000000000000000,34,NULL,0);
```

Comment:
We still want these in here, right? Or did you add these previously?

Reply:
Correct, I had originally had the Lower Jeuno decor set like this but decided to put everything into a module as it's more of a custom thing as opposed to the rest of the decorations. These particular entries were added where there was a gap in the npcid range (from 17780999 to 17781022).